scripts/ConnectBoxManage.sh

```diff
@@ -476,6 +476,13 @@ function set_ssid () {
     exit 1;
   fi
 
+  local ssid_length=`printf "%s" "$val" | wc -c`
+
+  if [[ $ssid_length -gt 32 ]]; then
+    echo "SSID must be 32 octets or less"
```

Comment:
I've implemented a basic error message dialog in the UI to relay the error message from the script to the user. Also removed the skip decorators